docgen/src/Getting_started.md

```diff
@@ -281,14 +281,14 @@ function Search() {
   return (
     <div className="container">
       <SearchBox />
-      <RefinementList attributeName="category" />
+      <RefinementList attribute="category" />
       <Hits hitComponent={Product} />
     </div>
   );
 }
 ```
 
-The `attributeName` props specifies the faceted attribute to use in this widget. This
+The `attribute` props specifies the faceted attribute to use in this widget. This
```

Comment:
I’m not sure if I agree with attribute, it sounds like the value 🤔

Reply:
It's a matter of taste I think, from the user perspective I think it's the same. But with `attribute` we are a bit more consistent with our plural form since we are using `attributes` for `<Breadcrumb>` & `<HierarchicalMenu>`.

Reply:
Good point! Let’s keep it as you changed

Reply:
I was surprised too but yes it's more aligned with attributes from other widget (that could also have been renamed attributesNames but that sounded weird this was our conclusion at the option naming meeting).

👍